doc/install.rst

```diff
@@ -196,9 +33,8 @@ with::
 
   $ sudo -H pip install --upgrade luma.lcd
 
-.. warning:: The default pip bundled with apt on Raspbian Jessie is really old, and can 
-   cause components to not be installed properly. Please ensure that **pip 9.0.1** 
-   is installed prior to continuing::
-   
+.. warning:: The default pip bundled with apt on Raspbian Jessie is really old,
```

Comment:
This warning can be removed, Jessie is ancient.

Reply:
Removed